core/vibes/soul/primitives/blog-post-card/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { clsx } from 'clsx';
+
+import { Image } from '~/components/image';
+import { Link } from '~/components/link';
+
+export interface BlogPost {
+  id: string;
+  author?: string | null;
+  content: string;
+  date: string;
+  image?: {
+    src: string;
+    alt: string;
+  };
+  href: string;
+  title: string;
+  className?: string;
+}
+
+export function BlogPostCard({ title, image, content, href, date, author, className }: BlogPost) {
+  return (
+    <Link
+      className={clsx(
+        'group max-w-full rounded-b-lg rounded-t-2xl text-foreground ring-primary ring-offset-4 @container focus:outline-0 focus-visible:ring-2',
+        className,
+      )}
+      href={href}
+    >
+      <div className="relative mb-4 aspect-[4/3] w-full overflow-hidden rounded-2xl bg-contrast-100">
+        {image?.src != null && image.src !== '' ? (
+          <Image
+            alt={image.alt}
+            className="transition-transform duration-500 ease-out group-hover:scale-110"
```

Comment:
We probably need to add a `object-cover` class here to prevent the image from distorting.

Reply:
@apledger Fixed in both PRs:

![image](https://github.com/user-attachments/assets/02089d7d-e698-45bd-ac7b-93a75583d007)